packages/@okta/vuepress-site/guides/custom-error-pages/use-macros/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Use Macros
+---
+The following macros contain the configuration parameters for certain page elements. These macros inject specific content or functionality automatically.
+
+The graphic highlights and numbers the macros in the html editor, and then displays the corresponding number on the error page for each macro and what text or image is affected.
+
+![Macros on Error Page](/img/custErrorPage.png "Macros on Error Page")
+
+#### <span v-pre>`{{orgName}}`</span> 
+
+This macro controls the org name title that appears on the error page's browser tab. You can make changes to this text by removing the macro and adding your own.
```

Comment:
Being pedantic, macros don't _control_anything; they _insert_ text. More importantly, they don't presume _where_ the macro is used. For example, I could use the `{{orgName}}` macro to insert the organization's name anywhere on the page (if I wanted to), not just the title.

I think it makes more sense to describe macros as

> This macro inserts the organization's name. For example: ...